docs/source/torch_cuda_memory.rst

```diff
@@ -8,6 +8,15 @@ at any point in time, and optionally record the history of allocation events tha
 The generated snapshots can then be drag and dropped onto the interactiver viewer hosted at `pytorch.org/memory_viz <https://pytorch.org/memory_viz>`_ which
 can be used to explore the snapshot.
 
+.. note::
+
+   The memory profiler and visualizer described in this document only have visibility into the CUDA memory that is
+   allocted and managed through the PyTorch allocator.  Any memory allocated directly from CUDA APIs will not be
+   visible in the PyTorch memory profiler.
+
+   NCCL (used for distributed communication on CUDA devices) is a common example of a library that allocates significant
```

Comment:
Shall we change "significant" to some word softer?

Reply:
nontrivial?